components/dashboard/src/projects/ProjectSettings.tsx

```diff
@@ -90,6 +90,18 @@ export default function () {
                 checked={!project.settings?.keepOutdatedPrebuildsRunning}
                 onChange={({ target }) => updateProjectSettings({ keepOutdatedPrebuildsRunning: !target.checked })}
             />
+            <h3 className="mt-12">Workspace Starts</h3>
+            <CheckBox
+                title={<span>Incrementally update from old prebuilds</span>}
+                desc={
+                    <span>
+                        Whether new workspaces can be started based on prebuilds that ran on older Git commits and get
+                        incrementally updated.
+                    </span>
+                }
+                checked={!!project.settings?.allowUsingPreviousPrebuilds}
+                onChange={({ target }) => updateProjectSettings({ allowUsingPreviousPrebuilds: target.checked })}
+            />
```

Comment:
**suggestion**: Things to consider before merging:
1. From what I understand, the persistence feature below affects both prebuilds and workspaces, but is it valuable or helpful to surface this to users?
2. Could it make sense to bundle this setting with the PVC setting below for simplicity, especially since the setting is currently not working as expected (https://github.com/gitpod-io/gitpod/issues/14159)?
3. Using an alternative description to make it easier for users to understand the benefit of this option.

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="project-settings-before" src="https://user-images.githubusercontent.com/120486/200294356-93357ebc-1ea2-41d6-886f-0493faca4d1c.png"> | <img width="1440" alt="project-settings-after" src="https://user-images.githubusercontent.com/120486/200295412-30d79b67-d26a-464b-98eb-2f156e5e9098.png"> |

Reply:
Let's fine tune the preferences on this follow up pr https://github.com/gitpod-io/gitpod/pull/14461

Reply:
OK!